### DISCUSSION TITLE
Oscillator and Modulation and IFX CC control

### DISCUSSION THREAD

**Discussion by DepreTux** [bangcorrupt/hacktribe#54] [Ideas]
As of now, the only way to change the OSC, Modulation and IFX parameters is on the front panel for each part.

Accessing them via MIDI CC would allow for more interoperability with other gear.

Thanks again!

**Comment by bangcorrupt** [comment]
I want to add motion sequences for these as well as MIDI control.  Also, saving the parameter values for every Osc/Mod/etc in the pattern file, instead of returning to the default value when the pattern is changed.  No guarantee I'll ever get it working.

**Reply by DepreTux** [reply]
> I want to add motion sequences for these as well as MIDI control. Also, saving the parameter values for every Osc/Mod/etc in the pattern file, instead of returning to the default value when the pattern is changed. 

That's good to know.

> No guarantee I'll ever get it working.

What you've done already is more than I thought possible.
When we get something like this for free we know there are no guarantees!
Thanks!

**Comment by ptowns** [comment]
Oh yes motion! motion controls over osc would be crazy! Elektron like. So obviously you can change the osc every trigger or 4 times a beat, I haven't tried this with with a ARP at more rapid frequencies. So I can confirm the osc responds to sysex and you can change it remotely. 
https://www.instagram.com/fake.work/p/CB1sSVWhYz6/?utm_medium=copy_link
Obviously this would be cool for glitchy sounds but oh how this could unlock some wavetable single cycle madness!
